templates/focus-ios.xliff

```diff
@@ -822,22 +825,42 @@
         <source>Learn More</source>
         <note>Text for the button to learn more about Tracking Protection.</note>
       </trans-unit>
+      <trans-unit id="trackingProtection.moreSettings">
+        <source>More Settings</source>
+        <note>Text for the button that closes the Shield Data and opens the regular Settings screen.</note>
+      </trans-unit>
       <trans-unit id="trackingProtection.socialTrackerLabel">
         <source>Social trackers</source>
         <note>label for social trackers.</note>
       </trans-unit>
+      <trans-unit id="trackingProtection.statusOff">
+        <source>Protections are OFF for this session</source>
+        <note>Text for the status off from Tracking Protection.</note>
+      </trans-unit>
+      <trans-unit id="trackingProtection.statusOn">
+        <source>Protections are ON for this session</source>
+        <note>Text for the status on from Tracking Protection.</note>
+      </trans-unit>
       <trans-unit id="trackingProtection.toggleDescription1">
         <source>Disable until you close %@ or tap ERASE.</source>
         <note>Description for the tracking protection toggle. Placeholder is either Firefox Focus or Firefox Klar</note>
       </trans-unit>
-      <trans-unit id="trackingProtection.toggleLabel">
-        <source>Tracking Protection</source>
-        <note>Text for the toggle that temporarily disables tracking protection.</note>
+      <trans-unit id="trackingProtection.toggleLabel2">
+        <source>Enhanced Tracking Protection</source>
+        <note>Text for the toggle that enableds/disables tracking protection.</note>
       </trans-unit>
-      <trans-unit id="trackingProtection.trackerDescriptionLabel">
-        <source>Choose whether %@ blocks ad, analytic, social, and other trackers.</source>
+      <trans-unit id="trackingProtection.trackerDescriptionLabel2">
+        <source>Keep your data to yourself. %@ protects you from many of the most common trackers that follow what you do online.</source>
         <note>General description of tracking protection settings, which is displayed underneath the trackers preferences in Settings. Placeholder is either Firefox Focus or Firefox Klar</note>
       </trans-unit>
+      <trans-unit id="trackingProtection.trackersBlockedLabel">
+        <source>Trackers blocked since </source>
```

Comment:
Shouldn't there be a colon at the end here?
Also, is the "since" needed? Seems to refer more to a date than to number of trackers blocked as is